variables.tf

```diff
@@ -388,6 +388,7 @@ variable "security_services" {
     disable_macie       = "false"
     disable_inspector   = "false"
     disable_securityhub = "false"
+    disable_stacksets   = "false"
```

Comment:
As configured it's not backwards compatible (fails if there is no `disable_stacksets` defined in the tfvars file). This in the variables.tf would solve that
```  
  type = object({
    disable_guardduty   = optional(bool, false)
    disable_macie       = optional(bool, false)
    disable_inspector   = optional(bool, false)
    disable_securityhub = optional(bool, false)
    disable_stacksets   = optional(bool, false)
  })
```